### DISCUSSION TITLE
Facing issue in postgres connectivity issue when generate private API.

### DISCUSSION THREAD

**Discussion by sivacmdiksha** [Sunbird-Knowlg/Community#156] [Bugs]
Hi Team ,

     In our dev environment, we have created the new Internal Kong pipeline in Data Pipeline. we have executed the boot swamp, provision kong db , Internal Kong pipeline.But we are facing issue in Onboard api's. We have tried many workarounds but nothing could helpful. Then we got to know that docker services are not running properly. Due to that we're facing postgres issue in Internal kong server. I have shared the error here. Can you please help us to proceed further on this issue? 
![image](https://github.com/Sunbird-Knowlg/Community/assets/126653605/23bc1883-b601-4ef9-aeae-67654f08f844)
![image](https://github.com/Sunbird-Knowlg/Community/assets/126653605/7d819d7e-29a6-4639-be1f-a5dfa29d2024)

**Comment by pallakartheekreddy** [comment]
@sivacmdiksha Thanks for reaching us. Please raise this request in Ed https://github.com/orgs/Sunbird-Ed/discussions

**Reply by pallakartheekreddy** [reply]
@sivacmdiksha Pls raise this in the ED discussion as requested. For now closing this discussion.
